src/main/groovy/com/webonise/userCrud/resources/CrudResource.groovy

```diff
@@ -0,0 +1,41 @@
+package com.webonise.userCrud.resources
+
+import com.google.inject.Inject
+import com.webonise.userCrud.pojos.UserPojo
+import com.webonise.userCrud.services.interfaces.DatabaseService
+import groovy.transform.CompileStatic
+import org.restlet.resource.Delete
+import org.restlet.resource.Get
+import org.restlet.resource.Post
+import org.restlet.resource.Put
+import org.restlet.resource.ServerResource
+
+@CompileStatic
+class CrudResource extends ServerResource {
+
+    @Inject
+    DatabaseService databaseService;
```

Comment:
What is DatabaseService? If you are playing with user then it should be UserService